packages/bot-handler-avs/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "dependencies": {
+    "@types/long": "4.0.1",
+    "@types/moment-duration-format": "2.2.2",
+    "@types/webrtc": "0.0.26",
+    "@wireapp/avs": "6.0.46",
+    "moment": "2.26.0",
```

Comment:
Mh, can we use date-fns here? (but it's not that important in a node env)

Reply:
mh, is it needed at all?

Reply:
Nope, my mistaked. Thanks for the hint.

Reply:
https://github.com/wireapp/wire-web-packages/pull/3110/commits/4eeba2f8533caa6080a5332e30e950022b8280d7
nice find, thanks.